addon/components/o-s-s/infinite-select.ts

```diff
@@ -11,10 +13,12 @@ interface InfiniteSelectArgs {
   searchPlaceholder: string;
   items: InfinityItem[];
   inline: boolean;
+  enableKeyboard?: boolean;
```

Comment:
Why would we not want the keyboard enabled ? 🤔

Reply:
InfiniteSelect is applied on other component than countrySelector too, on which keyboard controls are not yet active